### PR TITLE
Use phabricator client from libmozdata

### PR DESCRIPTION
**Pull request by La0**

### CODE REVIEW

**Comment by La0** [comment]
Depends on https://github.com/mozilla/libmozdata/pull/162

**Comment by La0** [comment]
@marco-c I released a new version of libmozdata, and use it here... Should be good to go if the build is green !

File: src/codecoverage/backend/codecoverage_backend/api.py [hunk 1]
```diff
@@ -5,10 +5,10 @@
```
Comment (marco-c) [inline src/codecoverage/backend/codecoverage_backend/api.py]: We could actually remove the Phabricator endpoint in codecoverage/backend, https://github.com/mozilla/release-services/issues/1992.

File: src/codecoverage/backend/requirements_frozen.txt [hunk 2]
```diff
@@ -48,14 +51,16 @@ flask-talisman==0.6.0
...
 future==0.17.1
 google-api-core==1.10.0
+google-api-python-client==1.7.7
 googleapis-common-protos==1.5.10
-google-auth==1.6.3
+google-auth==1.6.2
+google-auth-httplib2==0.0.3
 google-cloud-core==0.29.1
 google-cloud-storage==1.15.0
-google-cloud-storage=1.15.0
```
Comment (marco-c) [inline src/codecoverage/backend/requirements_frozen.txt]: ahah

File: src/codecoverage/backend/requirements_frozen.txt [hunk 3]
```diff
@@ -48,14 +51,16 @@ flask-talisman==0.6.0
 flit==1.3
 future==0.17.1
 google-api-core==1.10.0
+google-api-python-client==1.7.7
 googleapis-common-protos==1.5.10
-google-auth==1.6.3
+google-auth==1.6.2
```
Comment (marco-c) [inline src/codecoverage/backend/requirements_frozen.txt]: This was downgraded, expected?

File: src/codecoverage/backend/requirements_frozen.txt [hunk 4]
```diff
@@ -88,12 +94,15 @@ py==1.8.0
...
 pycodestyle==2.5.0
+pycparser==2.19
 pyflakes==2.1.1
 Pygments==2.3.1
+pyOpenSSL==19.0.0
 pytest==4.3.1
 pytest-asyncio==0.10.0
 pytest-cov==2.6.1
 pytest-runner==4.4
+python37==interpreter
```
Comment (marco-c) [inline src/codecoverage/backend/requirements_frozen.txt]: ?

File: src/codecoverage/backend/codecoverage_backend/api.py (hunk 1 above)
Comment (La0) [reply src/codecoverage/backend/codecoverage_backend/api.py]: I did a PR for that, and rebased this one on top (first 2 commits): no more `src/codecoverage/backend` changes here.

File: src/codecoverage/backend/requirements_frozen.txt (hunk 2 above)
Comment (La0) [reply src/codecoverage/backend/requirements_frozen.txt]: yeah... no comment.

File: src/codecoverage/backend/requirements_frozen.txt (hunk 4 above)
Comment (La0) [reply src/codecoverage/backend/requirements_frozen.txt]: Oups. My sed regex to build the frozen requirements was a bit too eager

File: src/codecoverage/backend/requirements_frozen.txt (hunk 3 above)
Comment (La0) [reply src/codecoverage/backend/requirements_frozen.txt]: Useless now since i removed ccov backend changes

**Comment by La0** [comment]
This will need a rebase once #2132 lands

**Pull request merged**